h2o-algos/src/main/java/hex/tree/isofor/IsolationForestModel.java

```diff
@@ -0,0 +1,95 @@
+package hex.tree.isofor;
+
+import hex.ModelCategory;
+import hex.ModelMetrics;
+import hex.genmodel.utils.DistributionFamily;
+import hex.tree.SharedTreeModel;
+import water.Key;
+import water.fvec.Frame;
+import water.util.SBPrintStream;
+
+
+public class IsolationForestModel extends SharedTreeModel<IsolationForestModel, IsolationForestModel.IsolationForestParameters, IsolationForestModel.IsolationForestOutput> {
+
+  public static class IsolationForestParameters extends SharedTreeModel.SharedTreeParameters {
+    public String algoName() { return "IsolationForest"; }
+    public String fullName() { return "Isolation Forest"; }
+    public String javaName() { return IsolationForestModel.class.getName(); }
+    public int _mtries;
+    public long _sample_size;
+
+    public IsolationForestParameters() {
+      super();
+      _mtries = -1;
+      _sample_size = 256;
+      _max_depth = 8; // log2(_sample_size)
+      _sample_rate = -1;
+      _min_rows = 1;
+      _min_split_improvement = 0;
+      _nbins = 2;
+      _nbins_cats = 2;
+      // _nbins_top_level = 2;
+      _histogram_type = HistogramType.Random;
+      _distribution = DistributionFamily.gaussian;
+    }
+  }
+
+  public static class IsolationForestOutput extends SharedTreeModel.SharedTreeOutput {
+    public long _max_path_length;
+    public long _min_path_length;
+
+    public IsolationForestOutput(IsolationForest b) { super(b); }
```

Comment:
bad indentation (not really a problem).

Reply:
will look into it

Reply:
looks okay in IntelliJ - indented using 2 spaces, 4 space in total for the inner class